### PR TITLE
docs: adding kommander open dashboard command

### PR DESCRIPTION
**Pull request by msdolbey**
## Jira Ticket

n/a

## Description of changes being made
I didn't know this kommander open dashboard command was a thing.
Totally worth adding this in the docs, methinks

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4134.s3-website-us-west-2.amazonaws.com/

## Checklist

- [x] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by makkes — APPROVED** [review]
Thanks for adding this, @msdolbey!!

File: pages/dkp/kommander/2.2/install/networked/index.md [hunk 1]
```diff
@@ -94,13 +94,21 @@ helmrelease.helm.toolkit.fluxcd.io/velero condition met
...
+This command will open your default browser and go to the URL of the Kommander web interface, and also print the username and password in the CLI.
+
+If you prefer not to open your browser immediately, you can also use the following command to retrieve the URL used for accessing Kommander's Web interface:
 
 ```sh
 kubectl -n kommander get svc kommander-traefik -o go-template='https://{{with index .status.loadBalancer.ingress 0}}{{or .hostname .ip}}{{end}}/dkp/kommander/dashboard{{ "\n"}}'
 ```
 
-Use the following command to access the Username and Password stored on the cluster:
+And, use the following command to access the Username and Password stored on the cluster:
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/install/networked/index.md:111]:
```suggestion
And use the following command to access the username and password stored on the cluster:
```

**Review by gvasquezvargas — APPROVED** [review]
Thanks for spotting this, Matt! I made a few little proposals.

File: pages/dkp/kommander/2.2/install/networked/index.md (hunk 1 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.2/install/networked/index.md]: Thanks! added in a new commit

File: pages/dkp/kommander/2.2/install/networked/index.md [hunk 2]
```diff
@@ -94,13 +94,21 @@ helmrelease.helm.toolkit.fluxcd.io/velero condition met
...
 ## Access Kommander Web UI
 
-When all the `HelmReleases` are ready, use the following command to retrieve the URL used for accessing Kommander's Web interface:
+When all the `HelmReleases` are ready, use the following command to open the Kommander dashboard in your browser:
+
+```sh
+kommander open dashboard
+```
+
+This command will open your default browser and go to the URL of the Kommander web interface, and also print the username and password in the CLI.
```
[the remark this answers is not in the document]
Comment (msdolbey) [reply pages/dkp/kommander/2.2/install/networked/index.md]: Thanks! added in a new commit

**Pull request merged**